kazoo/handlers/eventlet.py

```diff
@@ -13,6 +13,7 @@
 from eventlet.green import threading as green_threading
 from eventlet import queue as green_queue
 
+from kazoo.exceptions import KazooTimeoutError, TimeoutError
```

Comment:
As per https://github.com/python-zk/kazoo/pull/402#issuecomment-244782581 the goal is too allow one to keep importing TimeoutError from this module for compatibility.
If that is ok for you @bbangert , I can add an ignore mark on that error.